code/_globalvars/lists/flavor_misc.dm

```diff
@@ -173,7 +173,7 @@ GLOBAL_LIST_INIT(jumpsuitlist, list(PREF_SUIT, PREF_SKIRT))
 #define UPLINK_PDA "PDA"
 #define UPLINK_RADIO "Radio"
 #define UPLINK_PEN "Pen" //like a real spy!
-#define UPLINK_IMPLANT  "Implant"
+#define UPLINK_IMPLANT  "Implant (-4 TC)"
```

Comment:
This should use the actual cost of the uplink implant. Not certain if you want to handle that behavior in this define though..